src/apps/answers/tests/test_answers.py

```diff
@@ -3601,9 +3627,8 @@ async def test_validate_multiinformant_assessment_fail_target_subject_not_found(
 
         response = await client.get(url)
 
-        assert response.status_code == http.HTTPStatus.OK
-        assert response.json()["result"]["valid"] is False
-        assert response.json()["result"]["code"] == "invalid_target_subject"
+        assert response.status_code == http.HTTPStatus.UNPROCESSABLE_ENTITY
+        assert response.json()["result"][0]["message"] == "field required"
 
```

Comment:
this test should continue returning OK, you should pass all parameters, but a wrong target subject id